apps/native/app/src/graphql/queries/inbox.graphql

```diff
@@ -22,6 +22,20 @@ query ListDocuments($input: DocumentsV2DocumentsInput!) {
   }
 }
 
+query GetDocumentsSenders {
+  getDocumentSenders {
```

Comment:
I can get this info and categories along with the ListDocuments query but then when removing filters and loading the data again the selected senders/categories will be empty while loading so it is better to split this up like this :)